### ISSUE TITLE
Apply official scripting conventions

### ISSUE THREAD

**Issue by thebonbon** [EveronLife/EveronLife#52]
### Summarize your proposed improvement

As mentioned in https://github.com/EveronLife/EveronLife/discussions/51 the official scripting conventions and organization / coding styles should be applied to all existing scripts.

### Why is this needed?

This is part of the official scripting conventions and will keep everything organized and clean.
https://community.bistudio.com/wiki/Arma_Reforger:Scripting:_Conventions

**Comment by kojon1464** [comment]
I will be working on this

**Comment by DjCtavia** [comment]
There is something I've talked with rVn about **class constructor**.

Some files in Arma Reforger have some important logic code inside constructor, the problem is they're not open to modification since we can't mod a constructor.

Why not making a convention for that? Like **3 sub methods**: **PreConstruct, Construct, PostConstruct**.

**Comment by kojon1464** [comment]
I don't think that all 3 sub methods are needed. You can just have `InitClass()` method invoked in constructor. Moded class would have something like this:

```
override void InitClass()
{
    \\ PreConstructor code here
    super.InitClass();
    \\ PostConstructor code here
}
```

Quote from https://community.bistudio.com/wiki/Arma_Reforger:Scripting:_Conventions#modability: 
"_Constructor/Destructor: Modded class has its own const/destructor and cant modify parent one → you can have the constructor call another method like InitClass(_)"

Since this ticket is about offcial convention maybe discussion about our convention should be moved to some other discussion/issue.
